.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-python: "3.4"
+python: "3.5"
```

Comment:
Are you sure? Python 3.5.2 is the latest according to python.org, and 3.5.2 is what our Dockerfile uses anyways, so it's probably better that travis matches that than 3.4, which is what I have installed (outside docker) on my test machine.